pyproject.toml

```diff
@@ -64,9 +64,15 @@ packages = { "find" = { where = ["src"] } }
 [tool.isort]
 profile = "black"
 
+[tool.ruff]
```

Comment:
Can you please remove the configuration in this file for now-removed tools? e.g. `[tool.isort]` isn't necessary any more, right?